src/main/resources/io/jenkins/plugins/designlibrary/Links/index.groovy

```diff
@@ -17,32 +18,29 @@ def example(html) {
     }
 }
 
-namespace("/lib/samples").sample(title:_("title")) {
+
+
+s.sample(title:_("title")) {
     raw(_("blurb"))
 
     h2(_("define.title"))
     raw(_("blurb.define"))
+    s.code(language:"java", file:"Links.java")
 
     h2(_("breadcrumb.title"))
     raw(_("blurb.breadcrumb"))
 
 
     h2(_("hyperlink.title"))
     raw(_("blurb.modelLink"))
-    table(border:1) {
+    table(class:"jenkins-table") {
         example "<a href='.' class='model-link'>self</a>"
         example "<a href='..' class='model-link'>up</a>"
     }
 
     raw(_("blurb.modelLink.inside"))
-    table(border:1) {
+    table(class:"jenkins-table") {
         example "<a href='.' class='model-link inside'>self</a>"
         example "<a href='..' class='model-link inside'>up</a>"
     }
-
```

Comment:
The tl-tr seems to have no effect anymore since switching to tippy, so removing it.